### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by acidtone**
:wave:! GitHub Classroom created this pull request as a place for your teacher to leave feedback on your work. It will update automatically. **Don’t close or merge this pull request**, unless you’re instructed to do so by your teacher.

In this pull request, your teacher can leave comments and feedback on your code. Click the **Subscribe** button to be notified if that happens.

Click the **Files changed** or **Commits** tab to see all of the changes pushed to `main` since the assignment started. Your teacher can see this too.

<details>
<summary>
<strong>Notes for teachers</strong>
</summary>

Use this PR to leave feedback. Here are some tips:

- Click the **Files changed** tab to see all of the changes pushed to `main` since the assignment started. To leave comments on specific lines of code, put your cursor over a line of code and click the blue **+** (plus sign). To learn more about comments, read “[Commenting on a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)”.
- Click the **Commits** tab to see the commits pushed to `main`. Click a commit to see specific changes.
- If you turned on autograding, then click the **Checks** tab to see the results.
- This page is an overview. It shows commits, line comments, and general comments. You can leave a general comment below.

For more information about this pull request, read “[Leaving assignment feedback in GitHub](https://classroom.github.com/help/leaving-feedback-in-github)”.
</details>

Subscribed: @SANAZDANESHVAR

### CODE REVIEW

**Comment by acidtone** [comment]
Hi Sanaz, do you have a link to the original codepen this is based off of? Your code doesn't appear to include the javascript

File: index.html [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css" integrity="sha512-1PKOgIY59xJ8Co8+NE6FZ+LOAZKjy+KY8iq0G4B3CyeY6wYHN3yt9PW0XpSriVlkMXe40PTKnXrLnZ9+fkDaog==" crossorigin="anonymous" />
+  </head>
+  <title>codpen crazy</title>
+  </head>
+  <body>
+  <style>
```
Comment (acidtone) [inline index.html:11]: This code should be in a `link` element, not embedded in a `style` element.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  <div class="container">
+    <div class="input-field">
+      <i class="far fa-circle fa-1x" id="circle"></i>
+      <i class="fas fa-plus fa-1x" id="plus"></i>
+      <input type="text" id="input" placeholder="Add a new task" />
+    </div>
+
+  </div>
+</label>
+
```
Comment (acidtone) [inline index.html:103]: Was there no javascript in the codepen you used? That was a requirement of the bounty.

Comment (SANAZDANESHVAR) [reply index.html:103]: THANK YOU

**Pull request merged**